src/main/java/io/cdap/plugin/gcp/gcs/source/GCSSource.java

```diff
@@ -320,6 +337,11 @@ public boolean skipHeader() {
       return skipHeader == null ? false : skipHeader;
     }
 
+    @Nullable
+    public String getFileEncoding() {
```

Comment:
Just curious here, from https://github.com/cdapio/hydrator-plugins/pull/1255/files, I saw this method is only used in the validation, how does the input format knows which encoding is or am I missing anything

Reply:
@yaojiefeng The final piece is this PR in `core-plugins`. It's not part of the original PR as these are 2 modules with cross dependencies and I can't really build both of them on the same PR https://github.com/cdapio/hydrator-plugins/pull/1297